dex-it/src/test/scala/com/wavesplatform/it/sync/OrderHistoryTestSuite.scala

```diff
@@ -292,8 +294,8 @@ class OrderHistoryTestSuite extends MatcherSuiteBase with TableDrivenPropertyChe
 
         placeAndAwaitAtNode(mkOrderDP(mozart, wavesUsdPair, SELL, 95.waves, 2.0), isMarketOrder = isMarketOrder)
 
-        assertAvgWeighedPrice(mozart, List(288L))
-        assertAvgWeighedPrice(salieri, List(270L, 290L, 320L))
+        assertAvgWeighedPrice(mozart, List(288L  -> 259.20.usd))
+        assertAvgWeighedPrice(salieri, List(270L -> 135.usd, 290L -> 29.usd, 320L -> 96.usd))
```

Comment:
Interesting...
```scala
def executedPrice: Long = counter.price
```

So `salieri` spent USD as expected. But, how much USD should receive `mozart`? The same amount that `salieri` spent, isn't it?

Reply:
In other words: `259.20 != 260 = 135+29+96`